### PR TITLE
Add missing csrf_token field

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A
| --------------- | -----
| Branch?         | 1.2
| Bug fix?        | kind of
| New feature?    | no
| Related tickets | -

In a few of our plugins, we have included this change manually, so this PR fixes this.

### CODE REVIEW

File: src/Resources/views/orderRefunds.html.twig [hunk 1]
```diff
@@ -20,6 +20,7 @@
                     </div>
                 </div>
                 <form action="{{ path('sylius_refund_refund_units', {'orderNumber': app.request.attributes.get('orderNumber')}) }}" method="post">
+                    <input type="hidden" name="_csrf_token" value="{{ csrf_token(app.request.attributes.get('orderNumber')) }}" />
```
Comment (GSadee) [inline src/Resources/views/orderRefunds.html.twig:23]: Do we check this csrf token in action?

Comment (jakubtobiasz) [reply src/Resources/views/orderRefunds.html.twig:23]: Nope, thanks for noticing this. I don't know why we had this token in Plus, but I have added a validation now.

File: src/Action/Admin/RefundUnitsAction.php [hunk 2]
```diff
@@ -36,22 +38,34 @@ final class RefundUnitsAction
...
+    private CsrfTokenManagerInterface $csrfTokenManager;
+
     public function __construct(
         MessageBusInterface $commandBus,
         Session $session,
         UrlGeneratorInterface $router,
         RefundUnitsCommandCreatorInterface $commandCreator,
-        LoggerInterface $logger
+        LoggerInterface $logger,
+        CsrfTokenManagerInterface $csrfTokenManager,
```
Comment (GSadee) [inline src/Action/Admin/RefundUnitsAction.php:49]: As you were adding a new argument, you could use constructor property promotion here:
```suggestion
        private CsrfTokenManagerInterface $csrfTokenManager,
```

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Jakub! :tada:

**Pull request merged**